### ISSUE TITLE
[Device Type: Light] TypeError: Cannot set property 'temperatureK' of undefined

### ISSUE THREAD

**Issue by Chrischi-** [mikejac/node-red-contrib-google-smarthome#235]
<img width="370" alt="tempK" src="https://user-images.githubusercontent.com/6323217/149007361-fb413ba8-4715-481a-9faa-6cb2bb35b09c.png">

`
Color setting:
Color mode: Temperature K
`

I get the following error when creating a light device with Color Mode -> Temperature K

`TypeError: Cannot set property 'temperatureK' of undefined`

**Comment by Chrischi-** [comment]
@ckhmer1 

tried your PR #234 and got this:

```
Error: Type of color is object but it does not have matching property
    at DeviceNode.formatValue (/.node-red/node_modules/node-red-contrib-google-smarthome/devices/baseDevice.js:3001:23)
    at DeviceNode.setState (/.node-red/node_modules/node-red-contrib-google-smarthome/devices/baseDevice.js:3205:28)
    at /.node-red/node_modules/node-red-contrib-google-smarthome/devices/baseDevice.js:2906:24
    at Array.forEach (<anonymous>)
    at DeviceNode.cloneObject (/.node-red/node_modules/node-red-contrib-google-smarthome/devices/baseDevice.js:2902:35)
    at DeviceNode.updated (/.node-red/node_modules/node-red-contrib-google-smarthome/devices/baseDevice.js:2040:18)
    at GoogleSmartHome.execDevice (/.node-red/node_modules/node-red-contrib-google-smarthome/lib/Devices.js:208:36)
    at GoogleSmartHome._execDevice (/.node-red/node_modules/node-red-contrib-google-smarthome/lib/HttpActions.js:468:14)
    at GoogleSmartHome._exec (/.node-red/node_modules/node-red-contrib-google-smarthome/lib/HttpActions.js:373:50)
    at /.node-red/node_modules/node-red-contrib-google-smarthome/lib/HttpActions.js:209:28
    at Layer.handle [as handle_request] (/.node-red/node_modules/express/lib/router/layer.js:95:5)
    at next (/.node-red/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/.node-red/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/.node-red/node_modules/express/lib/router/layer.js:95:5)
    at /.node-red/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/.node-red/node_modules/express/lib/router/index.js:335:12)
    at next (/.node-red/node_modules/express/lib/router/index.js:275:10)
    at urlencodedParser (/.node-red/node_modules/body-parser/lib/types/urlencoded.js:82:7)
    at Layer.handle [as handle_request] (/.node-red/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/.node-red/node_modules/express/lib/router/index.js:317:13)
    at /.node-red/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/.node-red/node_modules/express/lib/router/index.js:335:12)
```

**Comment by ckhmer1** [comment]
This issue should be fixed, just waiting the new release

**Comment by Caprico85** [comment]
Released v0.2.3 containing @ckhmer1's fix.
